hnn_core/tests/test_viz.py

```diff
@@ -111,6 +113,27 @@ def test_dipole_visualization():
     # test plotting multiple dipoles with average
     fig = plot_dipole(dpls, average=True, show=False)
 
+    # test plotting dipoles with multiple layers
+    fig, ax = plt.subplots()
+    fig = plot_dipole(dpls, show=False, ax=[ax], layer=['L2'])
+    fig, axes = plt.subplots(nrows=3, ncols=1)
+    fig = plot_dipole(dpls, show=False, ax=axes, layer=['L2', 'L5', 'agg'])
```

Comment:
will this work if `axes` is not provided? ideally it should ...

Reply:
Indeed, but I guess probably we should not encourage users to do that? there's no reason to provide a list for `layer` without specifying the `axes`

Reply:
I see, you mean users should automatically get a subplots without specifying the axes object

Reply:
yes exactly :-) The `axes` argument is relevant if you want to add to the plots, for example to compare conditions